docs/getting-started.rst

```diff
@@ -22,6 +22,12 @@ Install and set up
 :doc:`installation-on-euler`.
 
 
+Download a test case study
+--------------------------
+
+Just go to this `link <https://github.com/architecture-building-systems/CityEnergyAnalyst/tree/master/cea/examples>`__.and unzip the case study where necessary.
```

Comment:
you know... you could also just do `cea extract-reference-case --destination ~/my/favorite/folder`...